cmake-ide.el

```diff
@@ -82,10 +82,34 @@
 
 (defconst cmake-ide-rdm-buffer-name "*rdm*" "The rdm buffer name.")
 
+(defvar cmake-ide-auto-complete-clang-enabled
```

Comment:
How are these new variables related to irony? Why disable functionality even if a package is installed?